modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+= Production Readiness Checklist 
+
+Before running production workloads on Redpanda it’s worth ensuring that the cluster is deployed and configured according to best practices. The checklist in this guide will help you ensure that you are set up for success. In this guide we identify three levels of production readiness - level 1 through level 3. At level 3 you will be ready to run mission critical workloads in the largest of enterprises, whereas smaller organizations or less critical workloads may suffice at level 1.
```

Comment:
```suggestion
Before running production workloads on Redpanda it’s worth ensuring that the cluster is deployed and configured according to Redpanda Data recommendations. The checklist in this guide will help you ensure that you are set up for success. In this guide we identify three levels of production readiness - level 1 through level 3. At level 3 you will be ready to run mission critical workloads in the largest of enterprises, whereas smaller organizations or less critical workloads may suffice at level 1.
```

Best practices varies from person to person. It's usually better to say that redpanda recommends those steps instead of just throwing best practices. Agree?

Reply:
@tmgstevens Critically important here are the definitions of each level. Do they align with the RP support level? If not, what is the guiding principle(s) that define each level. We need to clearly define what each Level number means.